packages/firestore/test/util/helpers.ts

```diff
@@ -285,6 +286,23 @@ export function patchMutation(
   if (precondition === undefined) {
     precondition = Precondition.exists(true);
   }
+  return patchMutationHelper(keyStr, json, precondition, /* updateMask */ null);
+}
+
+export function mergeMutation(
+  keyStr: string,
+  json: JsonObject<unknown>,
+  updateMask: FieldPath[]
+): PatchMutation {
+  return patchMutationHelper(keyStr, json, Precondition.none(), updateMask);
+}
+
+export function patchMutationHelper(
+  keyStr: string,
+  json: JsonObject<unknown>,
+  precondition: Precondition,
+  updateMask: FieldPath[] | null
+): PatchMutation {
```

Comment:
1. Does this have to be exported?
2. I don't like the name "helper". Can this be inline with patchMutation (with a default argument)